bn-apps/cordapp-updates-distribution/corda-updates-transport/src/main/kotlin/net/corda/cordaupdates/transport/CordaTransporterFactory.kt

```diff
@@ -0,0 +1,30 @@
+package net.corda.cordaupdates.transport
+
+import org.eclipse.aether.RepositorySystemSession
+import org.eclipse.aether.repository.RemoteRepository
+import org.eclipse.aether.spi.connector.transport.Transporter
+import org.eclipse.aether.spi.connector.transport.TransporterFactory
+import org.eclipse.aether.transfer.NoTransporterException
+
+/**
+ * Implementation of Maven Resolver's [TransporterFactory] for Corda transport.
+ *
+ * Corda transport expects a repository URL to be specified in the format of "corda:x500Name/repositoryName", i.e. for example "corda:O=BNO,L=New York,C=US/default".
+ * repositoryName allows a single node to serve artifacts from multiple repositories.
+ *
+ * TODO: replace '$' separator with ':' once it is confirmed that column can't be a part of x500 name
```

Comment:
Why not switch it around so the repository name comes first and the X.500 name after the slash? That way we're less vulnerable to an escaping issue if one day we allow / as a valid character in names

Reply:
Don't see how that would help. With the current URL formatting `corda:O=BNO,L=New York,C=US/default` the repo name is everything that goes after the last `/`. Even if we would allow `/` as a valid character in x500 names, the current parsing approach would still work, wouldn't it? 

Also this composition looks more logically consistent to me. Each next part complements the previous one. I.e. `protocol -> host -> repo`. While `protocol -> repo -> host` looks a bit broken as the repo name means nothing without the host in that context.